kong-0.14.1-0.rockspec

```diff
@@ -28,7 +28,7 @@ dependencies = {
   "luaossl == 20171028",
   "luasyslog == 1.0.0",
   "lua_pack == 1.0.5",
-  "lua-resty-dns-client == 2.1.0",
+  "lua-resty-dns-client == 2.2.0",
```

Comment:
It would have been nice to split the bump and the feature into 2 separate commits :)

On top of the commit atomicity benefits, we would likely avoid forgetting to update the documentation at release time (if you do not happen to do it by then), since a `feat(conf)` commit is quite visible when drafting the changelog.

Reply:
yes, good point. Adjusted it.